common/tests/threadpool_int/threadpool_int.c

```diff
@@ -409,7 +410,7 @@ TEST_FUNCTION(MU_C3(scheduling_, N_WORK_ITEMS, _work_with_pool_threads))
     execution_engine_dec_ref(execution_engine);
 }
 
-#define N_THREADPOOL_TIMERS 100
+#define N_THREADPOOL_TIMERS 1
```

Comment:
Do we want to have this as 1?  It seems like we'd want a least a few threads here.